runtime/string_functions.cpp

```diff
@@ -14,7 +14,7 @@
 #include "runtime/interface.h"
 
 const string COLON(",", 1);
-const string CP1251("1251", 4);
+const string CP1251("cp1251", 6);
```

Comment:
btw, it works fine like this too
```suggestion
const string CP1251("cp1251");
```

Reply:
done